node_modules/oae-preview-processor/lib/processors/link/default.js

```diff
@@ -86,6 +89,9 @@ var generatePreviews = module.exports.generatePreviews = function(ctx, contentOb
             ctx.addPreviewMetadata('embeddable', false);
         } else {
 
+            // If the Content-Disposition header is set to attachment the link isn't embeddable because the browser will prompt to save the target
+            var forcedDownload = response.headers['content-disposition'] && response.headers['content-disposition'].split(';')[0] === 'attachment';
```

Comment:
As mentioned before I would add a check for `Content-Type: application/octet-stream` here as well.